test/freetype_test.py

```diff
@@ -679,7 +679,7 @@ def test_freetype_Font_render_to(self):
         rrect = font.render_to(surf, rect, "FoobarBax", color, None, size=24)
         self.assertEqual(rrect.top, rrect.height)
         self.assertNotEqual(rrect.size, rect.size)
-        rrect = font.render_to(surf, (20.1, 18.9), "FoobarBax", color, None, size=24)
+        rrect = font.render_to(surf, (int(20.1), int(18.9)), "FoobarBax", color, None, size=24)
```

Comment:
I think the intention for this part of the test is to make sure floats are accepted (but will eventually be truncated to ints). The assert to test this (next line) is commented out so this might need some more investigation. There were similar deprecation warnings in `draw_test.py` and they were addressed by #1638. I would suggest undoing this change and maybe creating a new issue (or issues) to track if something similar to #1638 can be used to fix the deprecation warning and to figure out why the assert is commented out.

Reply:
Ah good call, 

I'll revert that part.